app/config.py

```diff
@@ -151,6 +151,7 @@ class Config(object):
     AWS_SES_ACCESS_KEY = os.getenv('AWS_SES_ACCESS_KEY')
     AWS_SES_SECRET_KEY = os.getenv('AWS_SES_SECRET_KEY')
     AWS_PINPOINT_REGION = os.getenv('AWS_PINPOINT_REGION', "us-west-2")
+    AWS_US_TOLL_FREE_NUMBER = os.getenv('AWS_US_TOLL_FREE_NUMBER', "us-west-2")
```

Comment:
Currently unsure if specifying the region here is necessary.

I also need to update K8s to hold the information here.

Reply:
Removed region here so as to simply pull the Toll Free Number